functions/src/endpoint_paths/point_log.ts

```diff
@@ -62,8 +62,8 @@ logs_app.post('/handle', async (req, res) => {
 		res.status(error.code).send(error.toJson())
 	} else {
 		try {
-			const should_approve = (req.body.approve === 'true');
-			const didUpdate = await updatePointLogStatus(should_approve, req.body.approver_id, req.body.point_log_id)
+			var should_approve = (req.body.approve == 'true');
```

Comment:
When declaring variables, Firebase's Linter requires us to declare all variables as either **const** (if the variable never changes or is a pointer to an object) or **let** (if the variable is a basic type whose value changes or is a pointer to an object that is reassigned)

Reply:
Firebase doesn't want us to use **var**.